source/languages/en/dataplatform/installing.md

```diff
@@ -61,26 +62,28 @@ If you are using or plan to use Spark, you must install Java 8 on **each node th
 
 If you are on Ubuntu, run the following to install Java 8:
 
-```Ubuntu
-$ sudo apt-get install software-properties-common
-$ sudo apt-get install python-software-properties
-$ sudo add-apt-repository ppa:webupd8team/java
-$ sudo apt-get update
-$ sudo apt-get install -y oracle-java8-installer
+```bash
+sudo apt-get install -y software-properties-common python-software-properties debconf-utils
```

Comment:
Why remove the $? (As written tone is hard, I want to say this is a completely earnest question out of curiosity.)